openmetadata-ui/src/main/resources/ui/src/components/MyData/CustomizableComponents/CustomizeMyData/CustomizeMyData.tsx

```diff
@@ -220,10 +209,22 @@ function CustomizeMyData({
     });
   };
 
-  const handleBackgroundColorUpdate = async (color: string) => {
+  const handleBackgroundColorUpdate = async (color?: string) => {
     await onBackgroundColorUpdate?.(color);
   };
 
+  const handleReset = useCallback(async () => {
+    // Get default layout with the empty widget added at the end
+    const newMainPanelLayout = getLayoutWithEmptyWidgetPlaceholder(
+      customizeMyDataPageClassBase.defaultLayout,
+      2,
+      4
+    );
+    setLayout(newMainPanelLayout);
+    await handleBackgroundColorUpdate();
+    await onSaveLayout();
```

Comment:
Do we need try catch here?

Reply:
No, they are already taken care of in the parent element where the API call is being made.